lisk-dex-electron/src/App.jsx

```diff
@@ -102,7 +102,7 @@ class App extends React.Component {
       this.assetAdapters[asset] = assetAdapter;
     }
 
-    await this.setState({
+    this.setState({
```

Comment:
The reason why there were await statements before `this.setState` is because there was some glitchy behavior which caused new orders to flicker but if you're confident that this is no necessary, I'm OK with this.

Reply:
@jondubois `this.setState` doesn't return a `Promise` it's a callback function nor does it need to return a `Promise`. React re-renders the components where the state changes. It does accept a callback function that's optional.

[here is a link with more in detail info](https://medium.com/@iamsongcho/is-setstate-async-b1947fbb25e5)

Reply:
I h encountered that the state was in loading... With an error. I'll verify everything just to be sure.

Reply:
I couldn't encounter this problem anymore. I tried reloading various times. But I'm pretty sure that this doesn't do anything. I do however saw that it fetches no data on the address `2.56.214.88:7011`. It does return a `200` status.